auth/providers/azure/options.go

```diff
@@ -90,6 +98,32 @@ func (o *Options) Validate() []error {
 	if o.TenantID == "" {
 		errs = append(errs, errors.New("azure.tenant-id must be non-empty"))
 	}
+
+	o.AuthzMode = strings.ToLower(o.AuthzMode)
+	switch o.AuthzMode {
+	case AKSAuthzMode:
+	case ARCAuthzMode:
+	case "":
+	default:
+		errs = append(errs, errors.New("invalid azure.authz-mode. valid value is either aks or arc"))
```

Comment:
```suggestion
		errs = append(errs, errors.New("invalid azure.authz-mode. valid value is either aks , arc, or empty"))
```
empty is also legit

Reply:
empty is default value but we don't want customers to set that. If customer does not choose that it will automatically set hence didn't mention in the valid values.